REUSE.toml

```diff
@@ -0,0 +1,25 @@
+version = 1
+
+SPDX-PackageName = "covfie, a part of the ACTS project"
+
+[[annotations]]
+path = [
+        "docs/**",
+        "README.md",
+        "extra/layers/README.md"
+]
+SPDX-FileCopyrightText = "2022 CERN"
```

Comment:
Can this be deduplicated by putting it outside of the `[[annotations]]` so that the same copyright text is used for both the docs and the code?

Reply:
Unfortunately not, in principle every section/annotation should have its own license/copyright statement.